ivy/functional/frontends/mindspore/ops/function/nn_func.py

```diff
@@ -36,3 +36,8 @@ def selu(input_x):
 @to_ivy_arrays_and_back
 def softsign(x):
     return ivy.divide(x, ivy.add(1, ivy.abs(x)))
+
```

Comment:
Please fix the lint/formatting issues with this PR as well: https://github.com/unifyai/ivy/actions/runs/5729438999/job/15526176909?pr=21018